content/api/events/events_post.md

```diff
@@ -1,35 +1,37 @@
 ---
 title: Post an event
 type: apicontent
-order: 10.1
+order: 11.1
 external_redirect: /api/#post-an-event
 ---
 
 ## Post an event
+
 This end point allows you to post events to the stream. Tag them, set priority and event aggregate them with other events.
 
 ##### ARGUMENTS
-* **`title`** [*required*]:  
-    The event title. *Limited to 100 characters.*  
+
+*   **`title`** [*required*]:
+    The event title. _Limited to 100 characters._
```

Comment:
Is there any reason we're changing from single `*` to `_`?

Reply:
yup, no reason to change this. I find it makes it more confusing to use different symbols in different places. they are both correct, but the asterisk is a bit more common.

Reply:
My code editor did those changes automatically when saving the `.md`, I'll try to remove those changes.